docs/5.0/application-access.md

```diff
@@ -0,0 +1,264 @@
+* Access to the Kubernetes Dashboard
+* Developer tools. Such as Jenkins, or the Atlassian stack.
+
+#### Hardware Requirements
+We recommend reviewing our [How it works](https://gravitational.com/teleport/how-it-works/) page to get a good overview of how Teleport works. You'll need a small VM or even a Raspberry Pi to run Teleport (Auth and Proxy).  This will be the brains and gateway to your applications.
+
+#### Networking Requirements
+
+!!! note "Why do I see SSH Ports for Teleport Application Access?"
+
+    Below is a list of ports required for Teleport Application Access to work but why SSH ports? The reason for this is Teleport uses reverse tunnels to obtain access to these applications.
+
+|Port      | Service    | Description
+|----------|------------|-------------------------------------------
+|3023      | Proxy      | SSH port clients connect to used for the reverse tunnel.
+|3024      | Proxy      | SSH port used to create "reverse SSH tunnels" from behind-firewall environments into a trusted proxy server.
+|3025      | Auth       | SSH port used by the Auth Service to serve its API to other nodes in a cluster.
+|3080      | Proxy      | HTTPS connection to authenticate `tsh` users and web users into the cluster. The same connection is used to serve a Web UI.
+
+#### TLS Requirements
+
+## Example Applications
+
+As outlined in our introduction Teleport Application Access has been designed to support two types of applications.
+
+**Example Legacy App**</br>
+A device such as an load balancer might come with a control panel, but it doesn't' have any auth and can only be access via a privileged network. These applications are supported and can extend access beyond your network.
+
+Other example legacy apps.
+
++ An internal admin tool
++ Control panel for networking devices
+
+**Example Modern App**</br>
+Teleport Application Access supports all modern applications, these could be built in-house or off-the-shelf software such as jenkins, Kubernetes Dashboard and Jupyter workbooks.
+
++ Kubernetes Internal Dashboard
++ Grafana
++ Jupyter notebooks
++ In-house Single Page Apps (SPA)
++ SPA with custom Json Web Token support (JWT)
+
+## Teleport Application Service Setup
+
+Teleport
+**Define `/etc/teleport.yaml`**
+
+| Variable to replace | Description  |
+|-|-|
+| `nodename` | Name of node Teleport is running on |
+| `auth_token` | Static Join Token |
+| `public_addr` | Public URL and Port for Teleport |
+| `https_key_file` | LetsEncrypt Key File ( Wildcard Cert )  |
+| `https_cert_file` | LetsEncrypt Key File ( Wildcard Cert ) |
+
+
+```
+teleport:
+  nodename: i-083e63d0daecd1315
+  data_dir: /var/lib/teleport
+  auth_token: 4c7e15
+  auth_servers:
+  - 127.0.0.1:3025
+auth_service:
+  enabled: "yes"
+  listen_addr: 0.0.0.0:3025
+  tokens:
+  - proxy,node,app:4c7e15
+ssh_service:
+  enabled: "false"
+proxy_service:
+  enabled: "yes"
+  listen_addr: 0.0.0.0:3023
+  web_listen_addr: 0.0.0.0:3080
+  tunnel_listen_addr: 0.0.0.0:3024
```

Comment:
if this is not set, 3024 will merge with 3080. Also do we really need all this setup in the first sample. Let's make the first sample very short and add full config with possible values at the end.